docs/running.rst

```diff
@@ -131,6 +131,24 @@ Options:
 \-\-release
    Disable all debugging features for :ref:`release`
 
+\-\-config-file
+  Read compiler configurations from a ``.toml`` file. The minimal fields required in the configuration file are:
+   
+  .. code-block:: toml
+
+    [package]
+    input_files = ["flipper.sol"]  # Solidity files to compile
+
+    [target]
+    name = "solana"  # Target name
+
+
+  Fields that are not explicitly present in the .toml file are defaulted. If any other argument is provided in the command line, for example, ``solang compile --config-file --target substrate``, the argument will be overridden.
```

Comment:
Do I need to specify the `.toml` file I am referring to when I run `solang compile --config-file`? What happens when I have more than one `.toml` file in my repository?

If you are creating a `.toml` with a reserved name, that should be noted in the documentation.

Reply:
The default one is `solang.toml` , but if a special name is used, it will be taken instead.